docs/usage/dashboards.md

```diff
@@ -1,21 +1,129 @@
 # Custom dashboards
 
-## Sharing and access restriction
+OpenCTI provides an adaptable and entirely customizable dashboard functionality. The flexibility of OpenCTI's dashboard ensures a tailored and insightful visualization of data, fostering a comprehensive understanding of the platform's knowledge, relationships, and activities.
 
-`Organizations`, `groups`, or `users` who have access to a dashboard can have 3 levels of access: 
-- `admin` read, write, access management
-- `edit` read and write
-- `view` read-only
 
-When a user creates a custom dashboard, it is only visible to themselves. They then have `admin` access. They can then define who can access it and with what level of rights via the `Manage access` button at the top right of the dashboard page.
+## Dashboard overview
 
+You have the flexibility to tailor the arrangement of [widgets](widgets.md) on your dashboard, optimizing organization and workflow efficiency. Widgets can be intuitively placed to highlight key information. Additionally, dynamically resize widgets from the bottom right corner based on the importance of the information, enabling adaptation to specific analytical needs. This technical flexibility ensures a fluid, visually optimized user experience.
```

Comment:
Additionally, **you can** or **you have the possibility to** ~~dynamically~~ resize widgets from the bottom right corner based on the importance of the information, enabling adaptation to specific analytical needs.